### PR TITLE
email verification prompt docs

### PR DESCRIPTION
**Pull request by aaguiarz**
https://docs-content-staging-pr-9208.herokuapp.com/docs/connections/azuread-adfs-email-verification

### CODE REVIEW

File: articles/connections/azuread-adfs-email-verification.md [hunk 1]
```diff
@@ -27,11 +27,28 @@ The Azure AD connection also has a **Use Common Endpoint** property. When it's e
...
 
 The new tenant setting is in the **Migrations** section, and it's called **Default to 'Email Verification' setting for Azure AD/ADFS connections**.
 
 When this setting is disabled, `email_verified` will always be `true` for Azure AD/ADFS connections. When enabled, it will use the 'Email Verification' setting at the connection level.
+
+## Email Verification Flow for Azure AD/ADFS connections
+
+If your application requires that the emails from Azure AD/ADFS connection's users are always verified, you can enable the **Enable email verification flow during login for Azure AD and ADFS connections** option in the tenant's Advanced Settings section.
+
+After the user authenticates for the first time, with an non-verified email, Auth0 will ask the user to verify their email by entering a one-time-use code that was sent to their email account:
```
Comment (sebadoom) [inline articles/connections/azuread-adfs-email-verification.md]:
```suggestion
After the user authenticates for the first time with an non-verified email, Auth0 will ask the user to verify their email by entering a one-time-use code that will be sent to their email account:
```

File: articles/connections/azuread-adfs-email-verification.md [hunk 2]
```diff
@@ -27,11 +27,28 @@ The Azure AD connection also has a **Use Common Endpoint** property. When it's e
...
+![](/media/articles/connections/azuread-adfs-email-verification.png)
+
+If the user completes this step, the `email_verified` field will be set to `true`, and users will not be prompted again for email verification, unless Azure AD or ADFS return a different email for the user.
+
+This new screen is rendered using the New Universal Login Experience, even if you have Universal Login configured to use the Classic Experience. To learn how to customize it, read the [Universal Login Customization documentation](/universal-login/customization-new). 
+
+To learn how to customize the email that is sent to users, check the [Verification Email template documentation](/email/templatesverification-email-using-code-)
+
+:::warning
+When Azure AD does not return an `email` claim, Auth0 maps the [Azure UserPrincipalName](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/plan-connect-userprincipalname) as the email. Given there's no guarantee that the UserPrincipalName has a mailbox, Auth0 will **NOT** display the email verification prompt, the user will have `email_verified` = `false`.
```
Comment (sebadoom) [inline articles/connections/azuread-adfs-email-verification.md]:
```suggestion
When Azure AD does not return an `email` claim, Auth0 maps the [Azure UserPrincipalName](https://docs.microsoft.com/en-us/azure/active-directory/hybrid/plan-connect-userprincipalname) as the email. Given there is no guarantee that the UserPrincipalName is a mailbox, Auth0 will **NOT** display the email verification prompt and the user will have `email_verified` = `false`.
```

File: articles/email/templates.md [hunk 3]
```diff
@@ -244,6 +244,10 @@ You can [configure a **Redirect To** URL](#configuring-redirect-to) to send the
 
 The target URL handler should be prepared to gracefully handle other possible messages as well. 
 
+### Verification Email (Using Code)
+
+Azure AD and ADFS connections support an [email verification flow](/connections/azuread-email-verification#email-verification-flow-for-azure-ad/adfs- connections) using a one-time-use code. If you enable tat feature, users will be prompted to enter the code before continuing the authentication flow.
```
Comment (sebadoom) [inline articles/email/templates.md]:
```suggestion
Azure AD and ADFS connections support an [email verification flow](/connections/azuread-email-verification#email-verification-flow-for-azure-ad/adfs- connections) using a one-time-use code. If you enable this feature, users will be prompted to enter the code before continuing the authentication flow.

```

**Review by sebadoom — COMMENTED** [review]
Grammar.

**Review by jeffreylees — APPROVED** [review]

**Pull request merged**